packages/fast-usdc-deploy/src/upgrade-evm-dests.core.js

```diff
@@ -57,28 +57,17 @@ const config = /** @type {const} */ ({
         version: 'ics20-1',
       },
     },
-    /** FIXME #11149 use real values */
     legibleDestinationOverrides: toExternalConfig(
-      harden({
-        'eip155:1': {
-          relay: USDC(1_000_000n),
-        },
-        'eip155:43114': {
-          relay: USDC(500_000n),
-        },
-        'eip155:10': {
-          relay: USDC(500_000n),
-        },
-        'eip155:42161': {
-          relay: USDC(500_000n),
-        },
-        'eip155:8453': {
-          relay: USDC(500_000n),
-        },
-        'eip155:137': {
-          relay: USDC(500_000n),
-        },
-      }),
+      harden(
+        /** @type {FeeConfig['destinationOverrides']} */ ({
+          'eip155:1': { relay: USDC(500_000n) },
```

Comment:
_low priority_

For readability, would it make sense to add comment next to each key what chain it corresponds to?

Reply:
I asked the same thing; @0xpatrickdev noted the names are in [cctp-chain-info.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/orchestration/src/cctp-chain-info.js) ; I'll add a cross-reference.